### DISCUSSION TITLE
How to configure the IP while running UE and GNB with USRP B210?

### DISCUSSION THREAD

**Discussion by LLiaang** [srsran/srsRAN_Project#119] [General Help]
Hello everyone!
I have trouble connecting UE to GNB with UERP B210 and I have created an [Issue#117](https://github.com/srsran/srsRAN_Project/issues/117). 
After reading the other Discussions and Issues, I think the configurations of IP are the most likely cause(Assuming I have no problem with installing). Including the NGAP in amf.yaml, GTPU in upf.yaml and amf: bind_addr in gnb.yaml.
So I sincerely want to know how to configure the IP address correctly.
I would appreciate it very much if anyone give me some advice!

**Comment by pgawlowicz** [comment]
Please follow this tutorial: https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html
